src/MissionManager/CameraSectionTest.cc

```diff
@@ -896,13 +896,6 @@ void CameraSectionTest::_testScanForStartVideoSection(void)
     QCOMPARE(_cameraSection->settingsSpecified(), false);
     visualItems.clear();
 
-    invalidSimpleItem.missionItem() = _validStartVideoItem->missionItem();
```

Comment:
This test is still valid. But should be changed to something like setParam2(VIDEO_CAPTURE_STATUS_INTERVAL + 1) to make sure it fails given new setting.

Reply:
This test is for reading back a camera section in from the vehicle such that it goes back into the camera section of a waypoint instead of a separate VIDEO_START_CAPTURE mission item. When the user is saying "Start Video" from the camera section combo they have no option to change the interval. It is fixed by the code.